adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go

```diff
@@ -259,3 +260,17 @@ func testCreateRoutesWithClustersWebsocket(t *testing.T, apiYamlFilePath string)
 // 	assert.Contains(t, pathLevelSandCluster.GetName(), contextExtensionMapPath["sandClusterName"],
 // 		"Sandbox Cluster mismatch in route ext authz context. (Path Level Endpoints)")
 // }
+
+func TestCreateRoutesWithClusters(t *testing.T) {
+
+	apiYamlFilePath := config.GetMgwHome() + "/../adapter/test-resources/envoycodegen/api.yaml"
+	apiYamlByteArr, err := ioutil.ReadFile(apiYamlFilePath)
+	assert.Nil(t, err, "Error while reading the api.yaml file : %v"+apiYamlFilePath)
+	apiJsn, conversionErr := utills.ToJSON(apiYamlByteArr)
+	assert.Nil(t, conversionErr, "YAML to JSON conversion error : %v"+apiYamlFilePath)
+	mgwSwagger := operator.GetMgwSwaggerWebSocket(apiJsn)
+	//routes, clusters, _ := envoy.CreateRoutesWithClusters(mgwSwagger, nil)
+	routes, clusters, _ := envoy.CreateRoutesWithClusters(mgwSwagger, nil)
+	t.Log(routes)
+	t.Log(clusters)
```

Comment:
What are these logs? Do we need these for tests ?

Reply:
removed logs and added assertions for those